src/main/java/com/bravo/user/dao/model/Profile.java

```diff
@@ -2,14 +2,11 @@
 
 import java.time.LocalDateTime;
 import java.util.UUID;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
+
 import lombok.Data;
 
 @Entity
-@Data
```

Comment:
Getters and Setters are provided by Lombok @Data annotation. Why did you remove it? https://projectlombok.org/features/Data